scapy/config.py

```diff
@@ -50,11 +50,13 @@
 if TYPE_CHECKING:
     # Do not import at runtime
     import scapy.as_resolvers
+    from scapy.data import NmapKnowledgeBase
     from scapy.packet import Packet
     from scapy.supersocket import SuperSocket  # noqa: F401
     import scapy.asn1.asn1
     import scapy.asn1.mib
 
+
```

Comment:
Please avoid unneeded change to preserve the code history as much as possible.

Reply:
ok